lib/api_client/connection/middlewares/request/logger.rb

```diff
@@ -47,6 +47,15 @@ def gather_response_debug_lines(response, taken_sec, debug_lines)
   end
 
   def current_stamp_millisec
```

Comment:
not the best idea of checking ruby version every time the method is called. I would rather define a method / pointer once during initialization of the gem based on the ruby version. This way it will be as fast as a method call without conditionals.